plash_cli/core.py

```diff
@@ -27,6 +27,7 @@ def get_client(cookie_file):
         raise FileNotFoundError("Plash config not found. Please run plash_login and try again.")
     cookies = Path(cookie_file).read_json()
     client.cookies.update(cookies)
+    client.headers.update({'X-PLASH': True})
```

Comment:
LGTM. I'm just wondering if we could also set this serverside so that it's automatically caught when the cookies are read here. And if that would be better because then we set the cookies in one place instead of two. So it will be easier to reason about what the cookies are.

Reply:
@RensDimmendaal not sure I understand your recommendation for the plash server. Perhaps we could discuss? Regardless, for the cli side, is this good to merge in your opinion?

Reply:
How would the plash client know what header to expect from the server?

Also, why not use a standard ('true' or '1' for X-PLASH header) instead of code?